### ISSUE TITLE
Space after title tags.

### ISSUE THREAD

**Issue by studersi** [dune73/test-4#9]
I think most Markdown processors require a space after the title tags.

```
### Title

###Title
```
### Title

###Title

**Comment by dune73** [comment]
Assuming my pwd is the root folder or the tutorials. What one-liner would you use to fix this problem in all the files?

**Comment by studersi** [comment]
This should do the trick:
```
find . -wholename "*tutorial-*/README.md" -exec sed -i -r "s/^(#{2,3})([[:alnum:]])/\1\ \2/" '{}' \;
```

**Comment by dune73** [comment]
That looks good - and more elegant than what I would have come up with. I have a tendency to make clumsy use of the tools..

Addressed in #71. Closing this.
